src/style.scss

```diff
@@ -33,6 +33,15 @@
     box-shadow: unset !important;
     outline: unset !important;
   }
+
+  &::part(wrapper) {
+    &:before {
+      border-bottom-color: transparent;
+    }
+    border-color: transparent;
+    background-color: unset;
+    box-shadow: none;
+  }
```

Comment:
This is off - the change should be line 29. 

From this:
```css
&::part(input) {
```

To This:

```css
&::part(wrapper) {
```

And then remove all of this new code